components/comments/comment-card.jsx

```diff
@@ -0,0 +1,30 @@
+import { convertToDates } from "../../utils/convertDate";
+
+const SingleComment = ({ comment }) => {
+  const date = convertToDates(comment.created_at);
+
+  return (
+    <section className="card">
+      <h3>{comment.author} said:</h3>
+      <p>{comment.body}</p>
+      <>
+        {date[0]}-{date[1]}-{date[2]},
+        {date[3] >= 0 && date[3] < 12 ? (
+          <>
+            {date[3]}:{date[4]}AM
+          </>
+        ) : (
+          <>
+            {date[3]}:{date[4]}PM
+          </>
+        )}
+      </>
+      <p>Votes: {comment.votes}</p>
+
+      <button aria-label="I like this">👍</button>
```

Comment:
I really like these thumbs up and thumbs down buttons, this will make for a nice user experience :) the aria label is a fantastic touch in terms of accessibility.